glue_jupyter/bqplot/profile/viewer.py

```diff
@@ -28,7 +28,7 @@ class BqplotProfileView(BqplotBaseView):
     _layer_style_widget_cls = ProfileLayerStateWidget
 
     tools = ['bqplot:home', 'bqplot:panzoom', 'bqplot:panzoom_x', 'bqplot:panzoom_y',
-             'bqplot:xrange']
+             'bqplot:xrange', 'bqplot:yrange']
```

Comment:
I don't see why you don't want this? I mainly added this here so I can test it, but if there is a way for me to add this just for the test and not here and you really don't like here, then please let me know.

Reply:
No this seems fine I think